packages/error-reporting/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "./src/index.js",
   "repository": "GoogleCloudPlatform/google-cloud-node",
   "scripts": {
-    "test": "nyc --exclude=\"fuzzer.js\" mocha ./test/unit/**/*.js",
+    "test": "nyc --exclude=\"fuzzer.js\" mocha ./test/unit/*.js ./test/unit/**/*.js",
```

Comment:
On my machine, without the `./test/unit/*.js`, the js files in the `./test/unit` directory are not being tested when `npm test` is run.